source/Calamari/Kubernetes/SetupKubectlAuthentication.cs

```diff
@@ -443,7 +439,7 @@ string CreateKubectlConfig()
 
             environmentVars.Add("KUBECONFIG", kubeConfig);
 
-            if (scriptSyntax == ScriptSyntax.Bash)
+            if (CalamariEnvironment.IsRunningOnNix || CalamariEnvironment.IsRunningOnMac)
```

Comment:
This was just so I don't need to use the `ScriptSyntax` because I don't have it!